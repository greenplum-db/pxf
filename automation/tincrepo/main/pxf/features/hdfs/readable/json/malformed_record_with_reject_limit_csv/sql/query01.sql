-- start_matchsubs
-- m/ +:/
-- s/( +):/\1|/
-- m/\+$/
-- s/\+$//
-- m/^NOTICE:  found \d+ data formatting errors.*/
-- s/.*/NOTICE:  Found 1 data formatting errors (1 or more input rows). Rejected related input data./
-- end_matchsubs
-- @description query01 for PXF HDFS Readable Json with malformed record test cases
SELECT *
FROM jsontest_malformed_record_with_reject_limit
ORDER BY id;
