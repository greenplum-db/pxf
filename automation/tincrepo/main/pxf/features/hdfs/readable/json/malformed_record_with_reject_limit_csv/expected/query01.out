-- start_matchsubs
-- m/ +:/
-- s/( +):/\1|/
-- m/\+$/
-- s/\+$//
-- end_matchsubs
-- @description query01 for PXF HDFS Readable Json with malformed record test cases
SELECT *
FROM jsontest_malformed_record_with_reject_limit
ORDER BY id;
NOTICE:  Found 1 data formatting errors (1 or more input rows). Rejected related input data.
           created_at           |         id         | text  | user.screen_name | entities.hashtags[0] | coordinates.coordinates[0] | coordinates.coordinates[1] 
--------------------------------+--------------------+-------+------------------+----------------------+----------------------------+----------------------------
 Fri Jun 07 22:45:02 +0000 2013 | 343136547115253761 | text1 | SpreadButter     | tweetCongress        |                            |                           
 Fri Jun 07 22:45:02 +0000 2013 | 343136547136233472 | text3 | NoSecrets_Vagas  |                      |                            |                           
(2 rows)

